dependencies.yaml

```
@@ -92,6 +99,18 @@ dependencies:
             packages:
               - cuda-nvcc
 
+  rapids_build_skbuild:
+    common:
+      - output_types: [conda, requirements, pyproject]
+        packages:
+          - rapids-build-backend>=0.3.0,<0.4.0.dev0
```

```suggestion
          - rapids-build-backend>=0.3.2,<0.4.0.dev0
```

Since this support is being added brand-new here, and since this project's build tool dependencies don't affect any other RAPIDS projects, I recommend going all the way to the latest version ([`0.3.2`](https://pypi.org/project/rapids-build-backend/#history)))